application/classes/Ushahidi/Validator/Notification/Create.php

```diff
@@ -0,0 +1,41 @@
+<?php defined('SYSPATH') OR die('No direct access allowed.');
+
+/**
+ * Ushahidi Notification Validator
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Ushahidi\Core\Tool\Validator;
+use Ushahidi\Core\Entity\ContactRepository;
+use Ushahidi\Core\Entity\SetRepository;
+
+class Ushahidi_Validator_Notification_Create extends Validator
```

Comment:
If we're allowing contact creation through the API, I suspect we should insist those contacts have an associated user id.